services/src/main/java/org/exoplatform/task/util/TaskUtil.java

```diff
@@ -1295,6 +1298,27 @@ public static String getResourceBundleLabel(Locale locale, String label) {
     return resourceBundleService.getResourceBundle(resourceBundleService.getSharedResourceBundleNames(), locale).getString(label);
   }
 
+  /**
+   * Gets platform language of current user. In case of any errors return null.
+   *
+   * @return the platform language
+   */
+  public static String getCurrentUserLanguage(String userId) {
+    try {
+
+      LocaleContextInfo localeCtx = LocaleContextInfoUtils.buildLocaleContextInfo(userId);
+      LocalePolicy localePolicy = ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(LocalePolicy.class);
+      String lang = null;
```

Comment:
should we initialize to default language instead of null ?
So that, in case of error, we return something instead of null